modules/backend/widgets/form/assets/js/october.form.js

```diff
@@ -83,8 +83,13 @@
             return this.fieldElementCache
         }
 
-        var form = this.$el,
-            nestedFields = form.find('[data-control="formwidget"] [data-field-name]')
+        var form = this.$el
+
+        if (!form) {
+            return;
+        }
```

Comment:
Can you explain under what conditions this triggers?

Reply:
When adding item, then removing one, then adding one again. The form element cannot be found. Don't know why yet

Reply:
It fixed this
![CleanShot 2019-11-05 at 22 32 33](https://user-images.githubusercontent.com/6506510/68248000-454d4e80-001c-11ea-811b-cd35ff5cd942.gif)

Reply:
I found the problem, you were removing the element that the event was triggered on before the change event could be fired on the containing form widget.